lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2266,6 +2266,42 @@ private static function get_block_nodes( $theme_json, $selectors = array() ) {
 		return $nodes;
 	}
 
+	private function gutenberg_get_preset_from_declaration( $declaration ) {
```

Comment:
This is a big function that has a lot of code duplicated from `get_svg_filters`. If we can't find a better way to get the preset then we should at least extract the common code from these two functions into a helper.